firebase/database.rules.json

```diff
@@ -143,6 +143,10 @@
           ".read": "auth.uid === $uid",
           ".write": "auth.uid === $uid"
         }
+      },
+      "userGroups": {
+        ".read": true,
```

Comment:
Can this be made readable only by logged in users? Something like `.read: auth != null`? (I have not tested this)

Reply:
Done!